CONTRIBUTING.MD

```diff
@@ -0,0 +1,60 @@
+# Contribution Guidelines
+
+First off, thank you for thinking of contributing to this project! 🥳
+
+The following is a set of guidelines for contributing to Ona Data. Following these guidelines helps to communicate that you respect the time of the maintainers and your intent to help develop this project.
+
+## How to contribute
+
+### 1. Report a bug 🐛
+
+Incase you have encountered any issue within the project, please make sure:
```

Comment:
For first word, I think you mean "In case", "incase" (without a space) is a synonym of "enclose"

Reply:
Changed this to 'In case'